integration

```diff
@@ -1194,6 +1194,7 @@
   "youdroid/home-assistant-gitea",
   "youdroid/home-assistant-gogs",
   "youdroid/home-assistant-sickchill",
+  "Youtilitics/home-assisstant",
```

Comment:
PR is not editable.

Reply:
Allow maintainers to edit it.

Reply:
Thanks. How do I do that?

Reply:
There is no option anywhere to allow maintainers to edit my PR. I think you would waste less time by telling me what you want me to do instead of telling me to RTFM.

Reply:
It looks like this option is simply not available because the source repo belongs to an org:

https://github.com/orgs/community/discussions/5634